src/stores/AccountController.ts

```diff
@@ -146,7 +146,7 @@ export const login = async (email: string, password: string) => {
     useAccountStore.setState({ loading: false });
   });
 };
-export async function logout() {
+export async function logout(onlyState = false) {
```

Comment:
I would suggest splitting out the `clearLoginState` logic to its own method if it turns out we need it. An optional flag is kind of hard to intuit, especially when the logic is inverted where `logout(false)` means to actually logout and `logout(true)` means not to.

Reply:
075a5d6bde7bc10485a74af46852c40e0e35bf9b